app/src/main/kotlin/org/stypox/dicio/skills/joke/JokeOutput.kt

```diff
@@ -0,0 +1,35 @@
+package org.stypox.dicio.skills.joke
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.dicio.skill.context.SkillContext
+import org.dicio.skill.skill.SkillOutput
+import org.stypox.dicio.R
+import org.stypox.dicio.io.graphical.Body
+import org.stypox.dicio.io.graphical.Subtitle
+import org.stypox.dicio.util.getString
+
+sealed interface JokeOutput : SkillOutput {
+    data class Success(
```

Comment:
If you know the API will always work, you can avoid having the sealed interface and just have `data class JokeOutput` instead. But I guess this is more extendible in the future, so it's fine to leave it like this too.